src/components/MovieCard/movieCard.scss

```diff
@@ -0,0 +1,26 @@
+@import './../../sass/abstracts/variables.scss';
+
+.movie-card {
+    width: 300px;
+    margin: 0 25px 30px;
+
+    &__image {
+        margin-bottom: 15px;
```

Comment:
![image](https://user-images.githubusercontent.com/47173992/109595566-63ab5780-7b47-11eb-9477-9751a5cb2c06.png)

To prevent cards elements disposition the simplest way is to set `width` and `height` to the img explicitly. But this may cause images distortion. Another way is to put img into the container and align it inside that container appropriately.

Reply:
fixed, thanks